packages/mdc-checkbox/_mixins.scss

```diff
@@ -102,7 +102,9 @@
   white-space: nowrap;
   cursor: pointer;
   vertical-align: bottom;
+}
 
+@mixin mdc-checkbox-background-selector-disabled_ {
```

Comment:
This doesn't seem like the right name for this mixin. Maybe dropping `selector-` is enough, but cc @advorak for his opinion.

Mixins with "selector" in the name are general-purpose mixins used to embed `@content` within a particular selector, which is not what this does.

Reply:
I see...I was just copying the names of the two mixins included in this one. How about `mdc-checkbox-disabled-background-base-fill-color`?

Reply:
`mdc-checkbox-disabled-color_($color)` seems a little closer to what we want.

Then, in `mdc-checkbox.scss`, you can pass `$mdc-checkbox-disabled-color` as the parameter value.

Reply:
After talking to @kfranqueiro, you don't need a `$color` param - you can leave it as-is and just rename the mixin to `mdc-checkbox-disabled-container-color_`.